src/child.rs

```diff
@@ -171,6 +170,28 @@ pub fn run(
     if exit.success() {
         return Ok(stdout);
     } else {
-        return Err(Error::cli(command_name, stdout.into(), stderr.into(), exit).into());
+        let mut msg = String::new();
+        if stdout.len() > 0 {
+            msg.push_str("\nstdout:\n");
+            for line in stdout.lines() {
+                msg.push_str("  ");
+                msg.push_str(line);
+                msg.push_str("\n");
```

Comment:
This is going to re-create the double-output thing, since the output is already re-printed as the command is run.

Reply:
Oops, indeed!